NAMESPACE

```diff
@@ -70,11 +71,13 @@ importFrom(dplyr,right_join)
 importFrom(grDevices,col2rgb)
 importFrom(grDevices,rgb)
 importFrom(jsonlite,fromJSON)
+importFrom(magrittr,"%>%")
```

Comment:
The pipe is also exported from `dplyr` (which is already a direct dependency) but in fact `dplyr` gets it from `magrittr` so I don't mind.